lesson02/controllers/UserController.js

```diff
@@ -1,10 +1,67 @@
+const { render } = require('ejs');
+const { response } = require('express');
 const express = require('express');
+const { validationResult } = require('express-validator');
+
+let formObj = function () {
```

Comment:
関数として保持しなくても良いように思いますが、いかがでしょうか？

Reply:
グローバル変数のまま、
ルーティング関数でどう初期化していいかわからないので、関数内のオブジェクトにしようと思います。
何か別のアプローチがあるのでしょうか？

```suggestion
  doPostRegister: function (req, res) {
    //入力フォームの値をバインド
    let form = {
      userName: req.body.userName,
      emailAddress: req.body.emailAddress,
      password: req.body.password,
      confirmPassword: req.body.confirmPassword,
    }
```

Reply:
> すみません、用語に不慣れで、この行のことですね。
> 
> let　formObj = function () {
> 
githubのコメントは、表示されているコードの一番下の行に対してコメントしているのが慣習となっておりますので、覚えておいてください^^

> グローバル変数のまま、
> ルーティング関数でどう初期化していいかわからないので、関数内のオブジェクトにしようと思います。
> このほかに別のアプローチがあるのでしょうか？

modelにuserClassを作成し、必要な箇所でインスタンス化するのはいかがでしょうか？

Reply:
貴重なアドバイスありがとうございます。

ES5とES6の書き方を混ぜるなと思っていましが、気にする必要ないんですね。
クラス化して扱います。